frontend/src/Series/Index/Table/SeriesIndexRow.js

```diff
@@ -413,6 +414,21 @@ class SeriesIndexRow extends Component {
               );
             }
 
+            if (name === 'releaseGroups') {
+              const joinedReleaseGroups = releaseGroups.sort((a, b) => a.localeCompare(b)).join(', ');
+
+              return (
+                <VirtualTableRowCell
+                  key={name}
+                  className={styles[name]}
+                >
+                  <span title={joinedReleaseGroups}>
+                    {joinedReleaseGroups}
```

Comment:
Like baker said this is going to overflow, definitely needs to truncate nicely. The title will have the full list.

Reply:
I did my best here. I'm not a frontend developer instead. 😅

![image](https://user-images.githubusercontent.com/5982912/152768366-5ec99f3f-fe22-4fb7-911c-31f35466f2ae.png)